src/API/Common/Time/Util.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\API\Common\Time;
+
+class Util
```

Comment:
Would prefer if we drop this class. It's currently only needed in `Zipkin\SpanConverter`. Not sure how useful the `nanosToXxx` methods are, some(/most?) use-cases might need rounded values instead of floored values.

If we want to keep the class: should probably add `nanosToSeconds` and `microsToNanos`.

Reply:
dropped